### PR TITLE
Add minapi

### PR DESCRIPTION
**Pull request by katiesavage**
## Use this template to add a workshop to this repo. Add the following items:

- [x ] Your workshop design document, filled in with some details about your proposed workshop including its title and the three stages in the designer delineated, using the `workshop-designer.md` file in the template.

- [x ] A `README.md` file that describes the steps of the workshop. 

- [x ] Powerpoint Slides for the workshop.

- [ ] The workshop's solution in the `/solution` folder.

- [ ] An embedded Train the Trainer video.

- [x ] A Call to Action to enhance the workshop with a direct line to Microsoft Learn by linking to an appropriate Learn module.

A reviewer will be assigned to the workshop and will work with you to shape it for production. Thank you for your contribution!

### CODE REVIEW

File: full/intro-minapi/README.md [hunk 1]
```diff
@@ -0,0 +1,323 @@
...
+**Snippet 2** : Above app.MapGet("/", () => "Hello World!");, add the following lines of code:
+```csharp
+app.UseSwagger();
+app.UseSwaggerUI(c =>
+{
+   c.SwaggerEndpoint("/swagger/v1/swagger.json", "Todo API V1");
+});
+```
+
+The preceding code enables middleware to serve the generated OpenAPI description as JSON content, enables middle to serve the Swagger UI elements, and specifies the OpenAPI description's endpoint.
```
Comment (jlooper) [inline full/intro-minapi/README.md:98]: "enables middle - is this missing a word?

File: full/intro-minapi/README.md [hunk 2]
```diff
@@ -0,0 +1,323 @@
...
+## Goals
+
+In this workshop, we will discuss *Minimal APIs*.
+
+| **Goal**              | **Create a fast, powerful API with .NET 6**                                   |
+| ----------------------------- | --------------------------------------------------------------------- |
+| **What will you learn**       | Learn the basics of creating a web API that links to a persistent database.                                        |
+| **What you'll need**          | [.NET 6 and VS Code (For Windows)](https://aka.ms/dotnet-coding-pack-win), [.NET 6 and VS Code (For macOS)](https://aka.ms/dotnet-coding-pack-mac) |
+| **Duration**                  | 1 hour                                                                |
+| **Slides** | [Powerpoint](Minimal%20APIs%20for%20Students.pptx)
```
Comment (jlooper) [inline full/intro-minapi/README.md:17]: let's call the slides 'slides.pptx' so that we don't get in trouble with spaces

File: README.md [hunk 3]
```diff
@@ -40,6 +40,7 @@ Welcome to the [Next Generation's Team Workshop Library](presentation.pptx), bui
...
 1️⃣ | [Build an App without Using any Code with Microsoft Power Platform](./full/power-platform-canvas-app/README.md) | Haimantika Mitra | 1 hour  | Build a no-code App using Power Platform | | Power Apps, Power Platform, [Canvas apps](https://docs.microsoft.com/learn/modules/build-app-solution/?WT.mc_id=academic-56577-hmitra) |
 1️⃣ | [Getting Started with Development Containers in VS Code](./full/using-dev-containers-vscode/README.md) | Ornella Altunyan | 1 hour | How to leverage development containers in Visual Studio Code to collaborate across machines and development environments. | | Git, [Use a Docker container as a development environment with Visual Studio Code](https://docs.microsoft.com/learn/modules/use-docker-container-dev-env-vs-code/?WT.mc_id=academic-55190-ornella) |
+1️⃣ | [Build a Minimal API with .NET 6](./full/intro-minapi/README.md) | Katie Savage | 1 hour | Build your first web API by building a Todo list. | | .NET 6, SQLite, [Build a web API with minimal API, ASP.NET Core, and .NET 6](https://docs.microsoft.com/learn/modules/build-web-api-minimal-api/) |
```
Comment (jlooper) [inline README.md:43]: I'm wondering whether this is a level 2, what do you think?

File: full/intro-minapi/README.md [hunk 4]
```diff
@@ -0,0 +1,323 @@
...
+## Next steps
+
+[Create web apps and services with ASP.NET Core, minimal API, and .NET 6](https://docs.microsoft.com/learn/paths/aspnet-core-minimal-api/)
+
+
+## Feedback
+
+Be sure to give [feedback about this workshop](https://forms.office.com/r/MdhJWMZthR)!
+
+[Code of Conduct](../CODE_OF_CONDUCT.md)
```
Comment (jlooper) [inline full/intro-minapi/README.md:322]: let's doublecheck this link

**Review by jlooper — CHANGES_REQUESTED** [review]
looks good! I will get it queued for QA if you can do a few small edits :)

File: README.md (hunk 3 above)
Comment (katiesavage) [reply README.md]: Agreed. I'll switch it to 2.

**Review by jlooper — APPROVED** [review]
looks good!

**Pull request merged**